src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -38,11 +38,12 @@ type HookValidatedInputPropTypes = TextInputProps &
     inputType?: 'textInput' | 'textArea';
   };
 
-type passwordPropTypes = TextInputProps & {
+interface PasswordInputProps extends TextInputProps {
```

Comment:
Why `interface` instead of `type`?

Reply:
it looks like you didnt update the pr with a lot of changes https://github.com/osbuild/image-builder-frontend/commit/a67760ce695d9443b50103897f007b1152cdf1e2